README.md

```diff
@@ -292,93 +292,8 @@ custom script](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/Even
 As a result, developers can use Observables without migrating tons of code on
 the platform, since it's an easy drop-in wherever you're handling events today.
 
-The proposed API shape is as follows:
-
-```js
-dictionary ObservableEventListenerOptions {
-  boolean capture = false;
-  boolean passive;
-};
-
-partial interface EventTarget {
-  Observable on(DOMString type, optional ObservableEventListenerOptions options);
-};
-
-// `SubscribeCallback` is where the Observable "creator's" code lives. It's
-// called when `subscribe()` is called, to set up a new subscription.
-callback SubscribeCallback = undefined (Subscriber subscriber);
-callback ObserverCallback = undefined (any value);
-
-dictionary Observer {
-  ObserverCallback next;
-  VoidFunction complete;
```

Comment:
super nit: Idiomatically, these are almost always specified as `next`, `error`, `complete`. Basically in order of importance to users.